client/me/purchases/cancel-privacy-protection/index.jsx

```diff
@@ -214,7 +217,7 @@ const CancelPrivacyProtection = React.createClass( {
 				</Card>
 			</Main>
 		);
-	}
+	},
```

Comment:
I get the idea but frankly I'm not a fan of all those trailing commas, especially in this particular location. We should aim at removing code when we can, not adding more.